content/en/users/online-storage/object-storage/_index.md

```diff
@@ -0,0 +1,191 @@
+the 'egicli' components is quite handy. For instance to setup the access to the
+SWIFT endpoint at IFCA-LCG2 via the Pilot VO (vo.access.egi.eu) the following is
+needed:
+
+```sh
+egicli endpoint projects --site IFCA-LCG2
+id                                Name                       enabled    site
+--------------------------------  -------------------------  ---------  ---------
+13c11c4073f4456fac7df84c4eb8f85b  VO:vo.nextgeoss.eu         True       IFCA-LCG2
+5eb8959a799240a98f4f303f5fbd80be  VO:dteam                   True       IFCA-LCG2
+9170e65775964a3ba6b18d83a2ad1968  eosc-hub.eu:d4science.org  True       IFCA-LCG2
+999f045cb1ff4684a15ebb338af69460  VO:vo.access.egi.eu        True       IFCA-LCG2
+f1d0308880134d04964097524eace710  VO:training.egi.eu         True       IFCA-LCG2
+```
+
+and then simply
+
+```sh
+eval "$(egicli endpoint env --site IFCA-LCG2 --project-id 999f045cb1ff4684a15ebb338af69460)"
+```
+
+Now the Openstack CLI can be used to perform operations on the SWIFT endpoint.
+
+Starting from listing the available containers(buckets):
+
+```sh
+ openstack container list
+
++------------------+
+| Name             |
++------------------+
+| egi_endorsed_vas |
++------------------+
+```
+
+Creating a new container:
+
+```sh
+openstack container create test-egi
+
++---------+-----------+------------------------------------------------------+
+| account | container | x-trans-id                                           |
++---------+-----------+------------------------------------------------------+
+| v1      | test-egi  | tx000000000000000000afc-005f845160-2bb3ed4-RegionOne |
++---------+-----------+------------------------------------------------------+
+```
+
+Creating a new Object by uploading a file:
+
+```sh
+ openstack object create test-egi file1.txt
+
++-----------+-----------+----------------------------------+
+| object    | container | etag                             |
++-----------+-----------+----------------------------------+
+| file1.txt | test-egi  | 5bbf5a52328e7439ae6e719dfe712200 |
++-----------+-----------+----------------------------------+
+```
+
+Listing objects inside a container:
+
+```sh
+ openstack object list test-egi
+
++-----------+
+| Name      |
++-----------+
+| file1.txt |
++-----------+
+```
+
+Download an object:
+
+```sh
+openstack object save test-egi file1.txt
+```
+
+Removing an object from the container:
+
+```sh
+openstack object delete test-egi file1.txt
+```
+
+Removing the entire container (-r option for recursive):
+
+```sh
+openstack container delete test-egi
+```
+
+### Access via S3 protocol
+
+Openstack SWIFT is compatible with S3 protocol, therefore if the SWIFT
+deployment are properly configured they can be accessed as any other S3
+compatible storage.
+
+In order to access the storage via S3 you need to create first the EC2
+credentials from the Openstack deployment.
+
+The following command is needed:
+
+```sh
+openstack ec2 credentials create
```

Comment:
cool, did you test this one? Is the obtention of ec2 credentials enabled by default? And is there anything needed at the swift side? Or just swift and S3 APIs are basically the same for most operations?

Reply:
the EC2 credentials creation is enabled by default, while the access via S3 protocol is not. So some configuration ( quite trivial ) is needed on the SWIFT side to enable S3

Reply:
Then there should be a way to discover whether the endpoint support S3 or not (I guess we could use GOCDB service type com.aws.s3). For the time being I think we can add a note about this being in progress or similar to avoid misunderstandings